client/src/reader/containers/notation/Picker.js

```diff
@@ -143,38 +144,46 @@ export class NotationPickerContainerImplementation extends PureComponent {
     } = this.composeProps(this.state.context, this.props);
 
     return (
-      <section role="search">
-        <div onMouseDown={this.handleMouseDown} role="presentation">
-          <Utility.Toggle
-            handleToggle={this.handleContextClick}
-            selected={this.state.context}
-            label="options"
-            optionOne={{
-              label: "resources",
-              icon: "resource24"
-            }}
-            optionTwo={{
-              label: "collections",
-              icon: "resourceCollection64"
-            }}
-          />
-          <EntitiesList
-            entityComponent={entityComponent}
-            entities={entities}
-            unit={singularUnit}
-            pagination={pagination}
-            callbacks={{
-              onPageClick: this.pageChangeHandlerCreator
-            }}
-            entityComponentProps={{
-              onRowClick: this.props.selectionHandler
-            }}
-            search={
-              <Search {...this.props.entitiesListSearchProps("notations")} />
-            }
-          />
-        </div>
-      </section>
+      <Translation>
+        {t => (
+          <section role="search">
+            <div onMouseDown={this.handleMouseDown} role="presentation">
+              <Utility.Toggle
+                handleToggle={this.handleContextClick}
+                selected={this.state.context}
+                label="options"
+                optionOne={{
+                  translatedLabel: t("glossary.resource_other"),
+                  label: "resources",
+                  icon: "resource24"
+                }}
+                optionTwo={{
+                  translatedLabel: t("glossary.collection_other"),
+                  label: "collections",
+                  icon: "resourceCollection64"
```

Comment:
See comment above. I think you can just handle the localization here.

Reply:
The label is used in class names and managing state, so we need that value without translation. Only the displayed label should be translated.

Reply:
Gotcha. Ideally, we'd solve this by doing something different with that `selected` prop, so we don't have string match to, e.g. "collections", especially since the toggle is only used in a couple of places. But you're right. Keep what you have.